src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -28,17 +28,26 @@ class AlternativesDialog < UI::Dialog
     MIN_WIDTH = 60
     MIN_HEIGHT = 20
 
+    def initialize
+      @mock_slaves = {
+        :ed => %!<pre>editor.1.gz /usr/share/man/man1/ed.1.gz</pre>!,
+        :vim => %!<pre>editor.1.gz /usr/share/man/man1/vim.1.
```

Comment:
in this case we prefer string format in a way ( and also please use new ruby symbol format ) and also there is a missing .gz here:
```
vim: "<pre>editor.1.gz /usr/share/man/man1/vim.1.gz" \
       "editor.fr.1.gz /usr/share/man/fr/man1/vim.1.gz" \
...
```